### ISSUE TITLE
[bench request]: Decision 2B and Decision Fast (FlyMyJev, native pointer probabilities)

### ISSUE THREAD

**Issue by denti** [fstandhartinger/jevbench#43]
Please evaluate these two frozen FlyMyJev research checkpoints on your infrastructure, including the held-out suite. The primary candidate is Decision 2B; Decision Fast is the smaller 0.6B variant. Both are runnable public weight packages with native trained-pointer probabilities and no submitter-operated endpoint.

| Candidate | Public weights, code and pinned revision | Parameters | Public accuracy (231 items) | Easy / standard / hard |
|---|---|---:|---:|---:|
| Decision 2B, v59 | [flymy-ai/decision-2b-preview](https://huggingface.co/flymy-ai/decision-2b-preview/tree/df57b75db927acc9ad91ec8115508c1e487086eb) | 2,275,545,600 | 174/231 = 75.32% | 100.00 / 83.33 / 59.46% |
| Decision Fast, v53a | [flymy-ai/decision-fast-preview](https://huggingface.co/flymy-ai/decision-fast-preview/tree/4225d41c66119fe28e95a2631bb0103decae6d56) | 606,672,384 | 148/231 = 64.07% | 100.00 / 77.78 / 39.64% |

These are developer measurements on the public subset, not an official rank or a full-suite score. We do not substitute an estimated judge tier for your held-out measurement.

**Interface and fixed mapping.** Both packages expose `model.load(...).decide(state, question)` and include `jevbench_adapter.py` plus `run_jevbench.py`, which uses your official Runner without changing its registry. Only state and the question's type, instructions and criteria enter inference. Choice labels and order are preserved; false/true maps to no/yes for noul; ordinal levels map to zero-based integer labels. No gold or task ID enters the model.

The readout is a trained pointer-head softmax, with each checkpoint's frozen temperature fitted on its own transfer replay and known-chance development set. It uses bf16, unmerged adapters and a4096-token packing limit. No answer generation, token-logprob readout, external model, permutation ensemble or retry-based selection is used. Earlier Fast measurements used token likelihood on77 long inputs; those results are not submitted here. These packages evaluate all items through the pointer head, without retraining either checkpoint.

**Run.** Linux, Python3.11, one CUDA GPU; tested on RTX4090 with Torch2.8.0+cu128, transformers4.57.6 and peft0.15.2. Install Torch, NumPy2.1.2 and the package's pinned requirements. Download into an ordinary local directory, rather than a symlink-based HF cache snapshot:

```python
from huggingface_hub import snapshot_download
snapshot_download("flymy-ai/decision-2b-preview", revision="df57b75db927acc9ad91ec8115508c1e487086eb", local_dir="decision-2b")
snapshot_download("flymy-ai/decision-fast-preview", revision="4225d41c66119fe28e95a2631bb0103decae6d56", local_dir="decision-fast")
```

```bash
git clone https://github.com/fstandhartinger/jevbench.git
git -C jevbench checkout 218511d85c5e9fc3dd12a08fae0e9818ea48bb68
python decision-2b/run_jevbench.py --harness ./jevbench --package ./decision-2b --output ./run-decision-2b --price-input-per-million 0.03
python decision-fast/run_jevbench.py --harness ./jevbench --package ./decision-fast --output ./run-decision-fast --price-input-per-million 0.01
```

Each package downloads its hash-pinned public base once, or accepts a local base using `--assets`. For private tasks use `--tasks /path/to/private.jsonl`; all inputs, outputs and model execution remain on your machine. The package also works through the typed Python interface if you prefer your own adapter. No hosted endpoint or ongoing GPU lease is needed from us.

**Cost and resources.** We have no production API tariff. The proposed nonzero hosted-reference estimates are $0.03/M input tokens for2B (the Qwen3.5-4B reference used for decider-2b) and $0.01/M for0.6B (the Qwen3-Embedding-0.6B reference used for Kev), with no generated output tokens. Please apply your final reference basis and latency adjustment. Local encoding+forward+calibration standard-tier p50/p95 was83.38/86.91ms for2B and59.78/60.70ms for Fast; network is excluded. Measured peak process RSS/allocated VRAM was5.08/4.83GiB and1.93/1.48GiB respectively. Requests are serial and bounded, with no response cache or external inference.

**Validation and disclosure.** The packaged2B reproduced all231 historical argmaxes; Fast reproduced all154 historical pointer argmaxes, with77 former likelihood items newly measured. Both repeated-request memory checks showed zero retained RSS/allocated-VRAM growth. Original validation reports preserve a too-strict0.51e-6 raw-probability check that flagged8/2 rows; the included numerical review accounts for six-decimal report rounding and float32 softmax normalization using one common scale bounded by two float32 epsilons. This numerical correction was made after the2B run and before Fast, with no change to weights, calibration or the zero-argmax-change requirement. We claim numerical compatibility at reported precision, not bitwise identity. The official Runner's three question-type mappings were also checked locally.

Public JevBench results informed earlier experiment design and candidate selection. Neither package claims benchmark-blind development or an independently proved complete absence of source overlap. Training source revisions, seeds, recipe hashes and original reports are included; a second end-to-end training run and fully public mixed-dataset reconstruction have not been established.

**Permissions and unresolved provenance.** These are evaluation previews. The author explicitly permits independent benchmark evaluation and publication of measurements in `EVALUATION-PERMISSION.md`; included code and separately pinned bases retain their Apache-2.0 notices. We do not assert a general commercial weight licence or complete training-source clearance. Both mixtures contain96 SNI task289/Gigaword-derived records whose project-specific upstream entitlement is not established, and source attribution for tasks1186/1283/1284 remains incomplete. The2B mixture also contains543 synthetic records labelled `claude-fable-5-1`, with applicable provider contract not established. These limitations are disclosed in each model card and `provenance-review.json`. Raw mixed training data and private evaluation cases are not published. If your eligibility policy requires broader weight licensing or resolved source rights, please mark the entry accordingly rather than treating this as a cleared commercial release.

Package manifests:2B `20d6a64318a9b865a7dd2a978fca049accfdce3a77b10105b9afd3b3b4799904`; Fast `7b5511b4a4e2473ad153298c8f1a1e7310098335072eaf272110f15091f06a3a`. HF file sizes, Git/LFS hashes and unauthenticated manifest access were verified after upload.

**Comment by fstandhartinger** [comment]
Thanks — this is logged for reachability checks and the JevBench measurement queue. We’ll only make benchmark claims after our own run.

- Harold
